docs/pom.xml

```diff
@@ -70,8 +70,27 @@ under the License.
     <!-- modules with documentation not need to install and deploy -->
     <maven.install.skip>true</maven.install.skip>
     <maven.deploy.skip>true</maven.deploy.skip>
+
+    <downloadDirectory>pom</downloadDirectory>
+    <downloadArtifactId>maven-parent</downloadArtifactId>
   </properties>
 
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-remote-resources-plugin</artifactId>
+        <executions>
+          <execution>
+            <id>site-download</id>
+            <phase>pre-site</phase>
```

Comment:
instead of putting this config into each pom leveraging the templates, I would rather set https://maven.apache.org/plugins/maven-remote-resources-plugin/process-mojo.html#skip based on the values of `downloadDirectory`/`downloadArtifact` in the main parent pom.

Reply:
in the most of we have a download page in project root pom, properties will also available in childs ...

Reply:
right, one possible way is to set the properties only for the current pom, but not if used as parent (https://github.com/apache/maven-apache-parent/blob/061e19af986a914df4d37b14e7e53a079d9033c5/pom.xml#L486)

Reply:
I think it will be more complicated ... we will need a mark file in all project